src/app/frontend/podlist/podcard_component.js

```diff
@@ -82,7 +82,24 @@ export class PodCardController {
    * @export
    */
   isFailed() {
-    return this.pod.podStatus.podPhase === 'Failed' || this.hasWarnings();
+    if (this.pod.podStatus.podPhase === 'Failed' || this.hasWarnings()) {
```

Comment:
I'm thinking about this check and I think it should be changed, because it will be true in case there are some warnings and pod is in running state. Pod state should be more important than events.

I think it should be: `if(podPhase === 'Failed' || (hasWarnings() && podPhase !== 'Running'))`

What do you think?